rhobs/alerting/data_plane/prometheus.oauth_alerts.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-spi-alerting-oauth
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: oauth_alerts
+      interval: 1m
+      rules:
+        - alert: OAuthLoginTime
+          expr: |
+            sum(rate(redhat_appstudio_spi_oauth_flow_complete_time_seconds_sum{type="GitHub"}[1h])) without(instance, pod)
```

Comment:
I don't see this metric in AppSRE Grafana.

Reply:
Hi, it's added recently:
https://github.com/redhat-appstudio/o11y/blob/main/dashboards/grafana-dashboard-rhtap-slos.configmap.yaml#L201

Reply:
That's the dashboard, but I don't see the metric that the alert and dashboard suppose to use.

Reply:
@yftacherzog is that you're asking for: https://gitlab.cee.redhat.com/rhtap/docs/sop/-/merge_requests/70/ ?

Reply:
Our metrics are sent to RHOBS and this is where the alerts are being evaluated. Both metrics and alerts are presented in AppSRE Grafana. If I go to production/staging data source in https://grafana.app-sre.devshift.net/, I get no output.

This is how you can troubleshoot this: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/prometheus/README.md#troubleshooting-missing-metrics-and-labels

Reply:
I think there were simply  no data (i.e. nobody did the oauth logins since pod restart). After i did couple of logins, data seems appeared:
![image](https://github.com/redhat-appstudio/o11y/assets/1651062/0c90ab6c-067a-4f22-946d-297d93bfd24b)

Is the `prometheus` label value seems correct to be collected into RHOBS?

Reply:
I see. alright!